### PR TITLE
feat(qwik-pod): filter function on issues page

### PR DESCRIPTION
**Pull request by Megio**
Added milestone and label filter on issue page

Fixes: #1149

### CODE REVIEW

File: qwik-graphql-tailwind/src/components/issue-tab-view/issue-tab-view.tsx [hunk 1]
```diff
@@ -45,22 +51,35 @@ export const IssueTabView = component$(({ activeTab, owner, name }: IssuesProps)
     activeTab: activeTab,
   });
   const dropdownStore = useStore<DropdownStoresProps>({
-    selectedLabel: labelOptions[0].value,
+    selectedLabel: undefined,
```
Comment (iamjerryhogan) [inline qwik-graphql-tailwind/src/components/issue-tab-view/issue-tab-view.tsx:54]: why undefined not null?

File: qwik-graphql-tailwind/src/components/issue-tab-view/issue-tab-view.tsx [hunk 2]
```diff
@@ -45,22 +51,35 @@ export const IssueTabView = component$(({ activeTab, owner, name }: IssuesProps)
...
     selectedSort: sortOptions[0].value,
-    selectedMilestones: milestonesOptions[0].value,
+    selectedMilestones: undefined,
```
Comment (iamjerryhogan) [inline qwik-graphql-tailwind/src/components/issue-tab-view/issue-tab-view.tsx:56]: why undefined and not null?

Comment (Megio) [reply qwik-graphql-tailwind/src/components/issue-tab-view/issue-tab-view.tsx]: in order to have the first "no milestone selected" value checked when we open the filter dropdown. We check the value of the option with the dropdown store value and the initial value of the store has no "selectedMilestone" so is computed as undefined, not null. (Don't know if I was clear, sorry)

File: qwik-graphql-tailwind/src/components/issue-tab-view/issue-tab-view.tsx (hunk 1 above)
Comment (Megio) [reply qwik-graphql-tailwind/src/components/issue-tab-view/issue-tab-view.tsx]: to be consistent with the milestone value, that needs to be undefined

**Review by SDaian — APPROVED** [review]
LGTM 🤘🏼

**Pull request merged**